README.md

```diff
@@ -30,14 +30,15 @@ $ docker run -d -p 80:80 -p 443:443 \
     -v /etc/nginx/vhost.d \
     -v /usr/share/nginx/html \
     -v /var/run/docker.sock:/tmp/docker.sock:ro \
+    --label jrcs.nginx_letsencrypt_companion.is_proxy=true \
```

Comment:
It should use a fully qualified domain name, i.e. `com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy=true`